src/pudl/metadata/resources/allocate_net_gen.py

```diff
@@ -0,0 +1,111 @@
+"""Resource metadata for the allocate_net_gen tables."""
+from typing import Any
+
+AGG_FREQS = ["yearly", "monthly"]
+
+RESOURCE_METADATA: dict[str, dict[str, Any]] = (
+    {
+        f"generation_fuel_by_generator_energy_source_{freq}_eia923": {
+            "description": (
+                f"{freq.title()} estimated net generation and fuel consumption "
```

Comment:
Should we describe what is estimated and how?

Reply:
Probably yes, but I think we should get that detail from @cmgosnell who is more familiar with the actual guts of the process.

Reply:
There is a tremendous amount of docs in the module. i think a lot of the module-level docs would be helpful to move over here to make it more visible to users.

Reply:
Do you know if the module level docs are in sync with what we're currently doing in the module?

Reply:
@cmgosnell Does the updated description I added make any sense? I tried to paraphrase the known drawbacks section as well.